code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm

```diff
@@ -909,91 +734,164 @@
-			name = "Waldo"
-			uniform = /obj/item/clothing/under/pants/jeans
-			suit = /obj/item/clothing/suit/striped_sweater
-			head = /obj/item/clothing/head/beanie/waldo
-			shoes = /obj/item/clothing/shoes/sneakers/brown
-			ears = /obj/item/radio/headset
-			glasses = /obj/item/clothing/glasses/regular/circle
-			back = /obj/item/storage/backpack/satchel/leather
-			backpack_contents = list()
-			if(prob(50))
-				backpack_contents += pickweight(list(
-					/obj/item/book/granter/spell/knock = 1,
-					/obj/item/book/granter/spell/blind = 1,
-					/obj/item/shadowcloak = 1,
-					/obj/item/book/granter/spell/smoke = 2,
-					/obj/item/reagent_containers/syringe/mulligan = 2,
-					/obj/item/dice/d20 = 3,
-					/obj/item/dice/d20/fate/stealth/one_use = 1,
-					/obj/item/clothing/head/chameleon/broken = 3,
-					/obj/item/stack/marker_beacon/ten = 3,
-					/obj/item/grenade/smokebomb = 3,
-					/obj/item/grenade/flashbang = 3
+				head = /obj/item/clothing/head/helmet/skull
+				suit = /obj/item/clothing/suit/armor/bone
+				gloves = /obj/item/clothing/gloves/bracer
+			if(prob(5))
+				back = pickweight(list(
+					/obj/item/spear/bonespear = 3,
+					/obj/item/fireaxe/boneaxe = 2
 					)
 				)
-			if(prob(50))
-				backpack_contents += pickweight(list(
-					/obj/item/book/granter/spell/knock = 1,
-					/obj/item/book/granter/spell/blind = 1,
-					/obj/item/shadowcloak = 1,
-					/obj/item/book/granter/spell/smoke = 2,
-					/obj/item/reagent_containers/syringe/mulligan = 2,
-					/obj/item/dice/d20/fate/stealth/one_use = 1,
-					/obj/item/dice/d20 = 3,
-					/obj/item/clothing/head/chameleon/broken = 3,
-					/obj/item/stack/marker_beacon/ten = 3,
-					/obj/item/grenade/smokebomb = 3,
-					/obj/item/grenade/flashbang = 3
+			if(prob(10))
+				belt = /obj/item/storage/belt/mining/primitive
+			if(prob(30))
+				r_pocket = /obj/item/kitchen/knife/combat/bone
+			if(prob(30))
+				l_pocket = /obj/item/kitchen/knife/combat/bone
+
+		if("Soldier")
+			mob_species = /datum/species/human
+			if(prob(90))
+				uniform = /obj/item/clothing/under/solgov
+				suit = /obj/item/clothing/suit/armor/vest/bulletproof/solgov
+				shoes = /obj/item/clothing/shoes/jackboots
+				gloves = /obj/item/clothing/gloves/color/black
+				mask = /obj/item/clothing/mask/gas/sechailer
+				head = /obj/item/clothing/head/solgov/sonnensoldner
+				id = /obj/item/card/id/solgov
+			else
+				uniform = /obj/item/clothing/under/solgov
+				suit = /obj/item/clothing/suit/space/hardsuit/solgov
+				shoes = /obj/item/clothing/shoes/combat
+				gloves = /obj/item/clothing/gloves/combat
+				mask = /obj/item/clothing/mask/gas/sechailer/swat
+				id = /obj/item/card/id/solgov
+			if(prob(85))
+				back = /obj/item/storage/backpack
+				backpack_contents = list()
+				for(var/i = 1 to 3)
+					if(prob(75))
+						backpack_contents += pickweight(list(
+							/obj/item/reagent_containers/hypospray/medipen/stimpack/traitor = 1,
+							/obj/item/storage/firstaid/tactical = 1,
```

Comment:
This has stimulants, medical night vision, and a combat defib in it. Change it to a conventional or medical kit, please.

Reply:
I didnt add this but i do agree its 2 powerful